site/src/pages/dashboard/[[...slugs]].page.tsx

```diff
@@ -63,7 +64,7 @@ const getDashboardSecondaryCardData = (
       title: "Generate Key",
       href: "/settings/api-keys",
     },
-    icon: "fa-solid fa-key",
+    icon: faKey,
```

Comment:
A small follow-up to https://github.com/blockprotocol/blockprotocol/pull/227

[Slack thread](https://hashintel.slack.com/archives/C02LG39FJAU/p1647270849625829) _(internal)_

Reply:
Thanks!